src/modules/landing_target_estimator/landing_target_estimator_params.c

```diff
@@ -187,3 +187,51 @@ PARAM_DEFINE_FLOAT(LTEST_SENS_POS_Y, 0.0f);
  *
  */
 PARAM_DEFINE_FLOAT(LTEST_SENS_POS_Z, 0.0f);
+
+/**
+ * Euler Angle Rotation (3-2-1 intrinsic Tait-Bryan) Phi (X)
+ *
+ * Orientation transformation from camera frame that detected
+ * the landing_target to drone's NED frame
+ *
+ * @unit rad
+ * @decimal 2
+ *
+ * @group Landing target Estimator
+ */
+PARAM_DEFINE_FLOAT(LTEST_EULR_PHI, 0.0f);
+
+/**
+ * Euler Angle Rotation (3-2-1 intrinsic Tait-Bryan) Theta (Y)
+ *
+ * Orientation transformation from camera frame that detected
+ * the landing_target to drone's NED frame
+ *
+ * @unit rad
+ * @decimal 2
+ *
+ * @group Landing target Estimator
+ */
+PARAM_DEFINE_FLOAT(LTEST_EULR_THETA, 0.0f);
+
+/**
+ * Euler Angle Rotation (3-2-1 intrinsic Tait-Bryan) Psi (Z)
```

Comment:
What's the difference with `LTEST_SENS_ROT` ?

Reply:
If we say that the precision landing sensor will always face downwards, I think we can achieve everything with `LTEST_SENS_ROT`.

I added the three angle parameters in the past because we are using data from a camera connected to an offboard system, that had an arbitrary rotation. I mostly wanted to allow users to input their data in an arbitrary reference frame, rather than expecting a specific one.